app/src/pages/playground/schemas.ts

```diff
@@ -219,3 +257,23 @@ export const llmToolSchema = z
   .optional();
 
 export type LlmToolSchema = z.infer<typeof llmToolSchema>;
+
+export const openAIResponseFormatSchema = z.lazy(() =>
```

Comment:
does this work with anthropic? Is the "strict" the only thing that is openai specific here? Just thinking about the naming and the unraveling we had to do before, anyway we can be a little more agnostic and allow for anthropic and openai in the same go in this pr?

Reply:
anthropic does not have response format as of now. this was intended to be the "hub" schema that we would add spokes off of, like anthropic in the future if necessary